wordguess.rb

```diff
@@ -0,0 +1,163 @@
+require 'colorize'
+require 'colorized_string'
+
+word_dictionary = ["valentine","cupid","heart","love","flowers","chocolate", "february", "roses", "sweetheart", "gifts", "dancing", "celebrate", "suitors", "forever", "promise", "flirt", "affection"]
+# start of word class
+class Word
+
+  attr_reader :current_word
+
+  def initialize (current_word)
+    @word_array = current_word.split("")
+    @all_guesses = []
+  end
+
+  # store guessed letter into an array
+  # checks word for the guessed letter
+  def compare_letter_to_word(guessed_letter)
+    @all_guesses << guessed_letter
+    @word_array.include?(guessed_letter)
+  end
+end
+# end of word class
+
+# start of game behaviors class
+class GameBehaviors
+
+  def initialize (word)
+    @word = word
+    @guess_counter = 5
+    @picture = [
+      " ,d88b.d88b, ",
+      " 88888888888 ",
+      " `Y8888888Y' ",
+      "   `Y888Y'   ",
+      "     `Y'     "]
+    @guess_status = []
+  end
+
+  def update_guess_counter(true_false)
+    @guess_counter -= 1 if true_false == false
+    return @guess_counter
+  end
+
+  def create_underscores
+    word_length = @word.instance_variable_get(:@word_array).length
```

Comment:
If you're going to get the word's `word_array` instance variable, use an `attr_reader` in the `Word` class.  In general, never use `instance_variable_get`.  That bypasses the public interface of the class.  

Also you're **never using** `word_length`